python/tvm/relay/op/contrib/dnnl.py

```diff
@@ -76,12 +81,83 @@ def make_pattern(with_bias=True):
         conv_out = is_op("add")(conv, bias)
     else:
         conv_out = conv
-    return is_op("nn.relu")(conv_out)
+    if with_eltwise:
+        return is_op(with_eltwise)(conv_out)
+    return conv_out
+
+
+def make_dense_pattern(with_bias=True, with_eltwise=None):
+    data = wildcard()
+    weight = wildcard()
+    bias = wildcard()
+    dense = is_op("nn.dense")(data, weight)
+    if with_bias:
+        dense_out = is_op("add")(dense, bias)
+    else:
+        dense_out = dense
+    if with_eltwise:
+        dense_out = is_op(with_eltwise)(dense_out)
+    return dense_out
+
+
+def make_dnnl_pattern(op, with_bias, with_eltwise):
+    pat_name = "dnnl." + op
+    pat_name += "_bias" if with_bias else ""
+    pat_name += ("_" + with_eltwise.split(".")[-1]) if with_eltwise else ""
+    if op == "conv2d":
+        return (pat_name, make_conv_pattern(with_bias, with_eltwise))
+    elif op == "dense":
+        return (pat_name, make_dense_pattern(with_bias, with_eltwise))
+    else:
+        return None
```

Comment:
suggest to add some warning msg like `Currently, only conv2d and dense op are supported, but got ()".format(op).`

Reply:
warning log has been added.